easybuild/easyconfigs/m/manta/manta-1.6.0.eb

```diff
@@ -0,0 +1,35 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:		https://staff.flinders.edu.au/research/deep-thought
+# Authors::		Robert Qiao <rob.qiao@flinders.edu.au>
+#
+# Software License::	GPLv3.0
+# Notes::		This is a binary build and only works on 64-bit linux
+##
+
+easyblock = 'CmdCp'
+
+name = 'manta'
+version = '1.6.0'
+
+homepage = 'https://github.com/arq5x/lumpy-sv'
```

Comment:
should'nt it be `https://github.com/Illumina/manta`?

Reply:
good spot, thx @migueldiascosta fixed  ✔️